docs/docs/what-about-generators.md

```diff
@@ -3,4 +3,63 @@ id: what-about-generators
 title: What About Generators?
 ---
 
-[what-about-generators](unfinished-article)
+[← Back To API Reference](/docs/api-reference.html)
```

Comment:
You can link to the actual api documentation by writing

`[Promise.coroutine](.)`